tests/fixtures/static_html/etrans_link.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>A page containing an eTrans button</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```

Comment:
The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

Reply:
These are fixtures for tests and the library in the Drupal website is included with defer as per https://github.com/openeuropa/oe_webtools/blob/master/oe_webtools.libraries.yml#L5